src/main/osd/osd.c

```diff
@@ -714,7 +739,7 @@ static bool osdDisplayStat(int statistic, uint8_t displayRow)
     case OSD_STAT_MAX_DISTANCE:
         if (featureIsEnabled(FEATURE_GPS)) {
             osdFormatDistanceString(buff, stats.max_distance, SYM_NONE);
-            osdDisplayStatisticLabel(displayRow, "MAX DISTANCE", buff);
+            osdDisplayStatisticLabel(displayRow, "MAX DIST", buff);
```

Comment:
Why abbreviating?

Reply:
To make space for the sticks value (8 characters reqd). Other option would be to change osdDisplayStatisticLabel() so that we don't require a fixed alignment of the ':' separator.

Reply:
(Was thinking about https://github.com/betaflight/betaflight/pull/10273)

Reply:
Ah, I guess I would need to look into moving the scrolling ticker into the osd code.